packages/react-intl/src/formatters/message.ts

```diff
@@ -72,6 +73,29 @@ function deepMergeFormatsAndSetTimeZone(
   };
 }
 
+export function patchFormatXMLElementFn(
```

Comment:
This is not really "patching", it neither mutates the input in-place, nor is it really what's happening under the hood. Can we figure out a better name?